Features/Context/SubContext/Authentication.php

```diff
@@ -35,7 +35,8 @@ public function goToPlatformUiAndLogIn($username, $password)
         $this->waitWhileLoading();
         $this->fillFieldWithValue('username', $username);
         $this->fillFieldWithValue('password', $password);
-        $this->iClickAtButton('Login');
+        $loginButton = $this->findWithWait('.ez-loginform-button');
```

Comment:
Nitpick, that also applies below: `$this->findWithWait('.ez-loginform-button')->click()`